content/patterns/radio/examples/radio-activedescendant.html

```diff
@@ -25,129 +25,129 @@
+            視覚的なキーボードフォーカスとホバーのスタイル設定には、CSS の <code>:hover</code> 及び <code>:focus</code> 擬似クラスを使用します。
             <ul>
-              <li>The focus indicator encompasses both the radio button and label, making it easier to perceive which option is being chosen.</li>
-              <li>Hover changes background of both the radio button and label, making it easier to perceive that clicking either the label or button will activate the radio button.</li>
-              <li>The cursor is changed to a pointer when hovering over the radio button to help people with visual impairments identify it as an interactive element.</li>
+              <li>フォーカスインジケータはラジオボタンとラベルの両方を囲み、どのオプションが選択されているかを認識しやすくします。</li>
+              <li>ホバーするとラジオボタンとラベルの両方の背景が変化し、ラベル又はボタンのいずれかをクリックするとラジオボタンがアクティブになることを認識しやすくします。</li>
+              <li>視覚障害のある利用者がインタラクティブな要素として識別できるように、ラジオボタンにカーソルを合わせるとカーソルがポインタに変わります。</li>
             </ul>
           </li>
           <li>
-            Because transparent borders are visible on some systems with high contrast enabled, only the focused element has a visible border.
-            When focusable elements are not focused, they have a 0-width border and additional padding equal in width to the border that is used to indicate focus.
+            ハイコントラストが有効になっている一部のシステムでは透明な境界線が表示されるため、フォーカスされた要素のみが可視境界線を持ちます。
+            フォーカス可能な要素がフォーカスされていない場合、それらは幅 0 の境界線と、フォーカスを示すために使用される境界線と同じ幅の追加のパディングを持ちます。
           </li>
           <li>
-            To ensure the inline SVG radio button graphics in the CSS file have sufficient contrast with the background when high contrast settings change the color of text content, CSS <code>forced-color-adjust</code> is set to <code>auto</code> on the <code>svg</code> graphics.
-            This causes the colors of the <code>stroke</code> and <code>fill</code> of the circle elements to be overridden by the high contrast color setting.
-            To make the background of the <code>circle</code> elements match the high contrast background color, the <code>fill-opacity</code> attribute of the outer <code>circle</code> is set to zero and the <code>stroke-opacity</code> attribute of the inner <code>circle</code> is set to zero.
-            If <code>forced-color-adjust</code> were not used to override the colors specified for the <code>stroke</code> and <code>fill</code> properties, those colors would remain the same in high contrast mode, which could lead to insufficient contrast between the radio circle elements and the background or even make them invisible if their color matched the high contrast mode background.<br>
-            Note: The explicit setting of <code>forced-color-adjust</code> is necessary because some browsers do not use <code>auto</code> as the default value for SVG graphics.
+            CSS ファイル内のインライン SVG ラジオボタングラフィックが、ハイコントラスト設定でテキストコンテンツの色が変更されたときに背景と十分なコントラストを持つことを保証するために、<code>svg</code> グラフィックの <code>forced-color-adjust</code> は <code>auto</code> に設定されます。
```

Comment:
修飾句の順序と欠落した語を補ってこんな感じですかね？
```suggestion
            ハイコントラスト設定がテキストコンテンツの色を変更したときに、CSS ファイル内のインライン SVG ラジオボタングラフィックが背景と十分なコントラストを持つことを保証するために、<code>svg</code> グラフィックの CSS <code>forced-color-adjust</code> は <code>auto</code> に設定されます。
```